pkgs/development/python-modules/black/default.nix

```diff
@@ -1,17 +1,17 @@
 { stdenv, buildPythonPackage, fetchPypi, pythonOlder
-, attrs, click, toml, appdirs, aiohttp, aiohttp-cors
+, attrs, click, toml, appdirs, aiohttp, aiohttp-cors, mypy-extensions, typing-extensions 
```

Comment:
Probably worth sorting these too and having them on separate lines. See comment below. Note that not ALL inputs need sorted, typically just the python ones.